libdnf/dnf-sack.c

```diff
@@ -1088,6 +1088,11 @@ dnf_sack_add_cmdline_package(DnfSack *sack, const char *fn)
         return NULL;
     }
     p = repo_add_rpm(repo, fn, REPO_REUSE_REPODATA|REPO_NO_INTERNALIZE);
+    if (p == 0) {
+        g_warning ("failed to read RPM: %s, skipping",
```

Comment:
I think it doesn't matter (also we don't have `GError` in this function and I don't want to break API/ABI)